src/components/ItemTable/ItemAvailabilityComponents/ContactALibrarian.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from "@nypl/design-system-react-components"
+import { useContext } from "react"
+import { FeedbackContext } from "../../../context/FeedbackContext"
+import type { ItemMetadata } from "../../../types/itemTypes"
+import type Item from "../../../models/Item"
+
+const ContactALibrarian = ({ item }: { item: Item }) => {
+  const { onOpen, setItemMetadata } = useContext(FeedbackContext)
+  const onContact = (metadata: ItemMetadata) => {
+    setItemMetadata(metadata)
+    onOpen()
+  }
+  return (
+    <>
+      {" - Please "}
+      <Button
+        id="contact-librarian"
+        buttonType="link"
```

Comment:
This reminds me that I wanna get an unstyled button variant into the DS. Will bring this up with Marty.

Reply:
Yes, this came up in the context of the error message and the button in the banner in your PR so make sure to include that. 

In this case, this `ContactALibrarian` button should not care where it's rendered, so the update should probably be made in the banner component.